pdfkit/pdfkit.d.ts

```diff
@@ -127,6 +127,9 @@ declare namespace PDFKit.Mixins {
         strike?: boolean;
         /**whether the text segment will be followed immediately by another segment. Useful for changing styling in the middle of a paragraph. */
         continued?: boolean;
+
+        /** the alignment of the text (center, justify, left, right) */
+        align?: any;
```

Comment:
thanks for your changes, everything looks good. 
would change `any` to `string`.
Cheers.